plugins/amoeba/platforms/reference/src/SimTKReference/AmoebaReferenceVdwForce.cpp

```diff
@@ -244,6 +244,16 @@ double AmoebaReferenceVdwForce::calculatePairIxn(double combinedSigma, double co
     double energy = combinedEpsilon * t1 * t2min;
     double dEdR = combinedEpsilon * (dt1 * t2min + t1 * dt2) / combinedSigma;
 
+    if (usesLJ) {
+        double pp1 = combinedSigma / r_ij;
+        double pp2 = pp1 * pp1;
+        double pp3 = pp2 * pp1;
+        double pp6 = pp3 * pp3;
+        double pp12 = pp6 * pp6;
+        energy = combinedEpsilon * (pp12 - 2.0 * pp6);
+        dEdR = combinedEpsilon * (pp12 - pp6) * (-12.0) / r_ij;
+    }
```

Comment:
This isn't the correct form for Lennard-Jones.  It should be `4*epsilon*((sigma/r)^12 - (sigma/r)^6)`.  The function used here is what you would use if you were specifying the distance by rmin rather than sigma.

Reply:
Well... Tinker uses Rmin instead of sigma as it's internal size descriptor for L-J, and for all of the two-parameter "Vdw" functions. In fact, the "sigma trick" only works for L-J, so using the Rmin values is in some sense more general. (In other words, only L-J has a "sigma", but essentially all two-parameter Vdw forms have an "Rmin".) The code in the pull request is correct for use in Tinker-OpenMM, where the parameters are passed in from Tinker. But it all depends on where the parameters will be coming from.

Reply:
Sigma and rmin are both fine ways of parametrizing LJ.  But the variable is called `combinedSigma`, and all the public functions in AmoebaVdwForce likewise refer to it as sigma.  If we ask the user to specify sigma but then interpret it as rmin instead, the result will be wrong.

Reply:
Yes, agreed. The variable should be called combinedRmin or something similar, and any other such references to Sigma should be Rmin, as that is really very confusing. Again Tinker does always use Rmin internally as, for example, there is no clean definition of a "sigma" for the buffered 14-7 function in AMOEBA. Sorry for the confusion.

Reply:
I don't think we can rename it.  The public API has always referred to it as sigma, and also has methods like `setSigmaCombiningRule()`.  We can't change those without breaking compatibility.  So I'll keep those as they are and treat it as being sigma for LJ.  Just remember to scale your rmin values by 2^(-1/6) to convert them to sigmas when calling `addParticle()`.

Reply:
Sure, that's fine. I keep forgetting that the AMOEBA plugin is not standalone code :) Just do whatever makes sense to you in the broader context of OpenMM, and we'll make the changes in our interface code, etc. to match. Thanks for looking at this carefully, and for all your help.